workflow/libero_flow_poc/docker-compose.yml

```diff
@@ -0,0 +1,88 @@
+version: '3'
+
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    command: >
+            /bin/bash -c "
+                until echo > /dev/tcp/postgres/5432; do sleep 1; done
+                python manage.py migrate && python manage.py runserver 0.0.0.0:8000
+            "
+    env_file:
+      - '.env'
+    ports:
+      - '8000:8000'
+    depends_on:
+      - postgres
+      - redis
+      - rabbitmq
+
+  decider:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    working_dir: /srv/app
+    command: >
+            /bin/bash -c "
+                until echo > /dev/tcp/rabbitmq/5672; do sleep 1; done
+                python run_decider.py
+            "
+    depends_on:
+      - rabbitmq
+
+  scheduler:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    working_dir: /srv/app
+    command: >
+            /bin/bash -c "
+                until echo > /dev/tcp/rabbitmq/5672; do sleep 1; done
+                python run_scheduler.py
+            "
+    depends_on:
+      - rabbitmq
+
+  worker:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    working_dir: /srv/app
+    command: >
+            /bin/bash -c "
+                until echo > /dev/tcp/rabbitmq/5672; do sleep 1; done
+                python run_worker.py
+            "
+    depends_on:
+      - rabbitmq
+
+  redis:
+    image: 'redis:3.0-alpine'
+    ports:
+      - '6379:6379'
+    volumes:
+      - 'redis:/data'
+
+  rabbitmq:
```

Comment:
Just pointing out that this RabbitMQ seems to be internal to the project, rather than being used to communicate with other components. Therefore it doesn't need to conform to the ADRs about the message bus (for example the message schema), though the fact that it is RabbitMQ makes it easier to operate since its setup will be similar to what is used in the message bus itself.

Reply:
The possible benefit of using the global bus would be other things could react to the events, but yes by design at present this has its own.

Reply:
The more this is hidden, the easier it is to change without other projects possibly breaking

Reply:
Yes good point.